db/seeds.rb

```diff
@@ -1,7 +1,62 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require File.expand_path('../../lib/seed_helper.rb', __FILE__)
+require 'csv'
+require 'open-uri'
+require 'colorize'
+
```

Comment:
This looks awesome!  I think we should add steps to clear the database first though, as after running this I now have 1022 trains in my dev db. 😅